package.json

```diff
@@ -22,13 +22,17 @@
     "@angular/platform-browser": "~8.2.4",
     "@angular/platform-browser-dynamic": "~8.2.4",
     "@angular/router": "~8.2.4",
+    "@ngrx/effects": "^8.6.0",
+    "@ngrx/store": "^8.6.0",
+    "@ngrx/store-devtools": "^8.6.0",
     "font-awesome": "^4.7.0",
+    "ngrx": "0.0.1-security",
```

Comment:
what is this package used for ?

Reply:
https://www.npmjs.com/package/ngrx/v/0.0.1-security 

this is a dummy package in order to avoid malicious use